recipes-core/niacctbase/niacctbase/sudoers

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
Confirming that this file is installed to a niacctbase namespaced directory.

Reply:
On-target? It gets [installed](https://github.com/ni/meta-nilrt/pull/336/files#diff-01422a91d9db4c848e9e979398fc25e4c9620da8553ab3dd96c836b1a3f64692R51) as `/etc/sudoers.d/niacctbase` (which I confirmed in the original round of testing.)